src/plugins/intel_cpu/src/cpu_streams_calculation.hpp

```diff
@@ -15,10 +15,13 @@ namespace ov {
 namespace intel_cpu {
 /**
  * @brief      Generate streams information table according to processors type table
- * @param[in]  input_streams is target streams set by user via NUM_STREAMS or hints.
+ * @param[in]  input_streams is target streams set by user via ov::num_streams or hints.
```

Comment:
input_streams is the targeted number of sreams set by user...

Reply:
updated